portfolio/src/main/webapp/script.js

```diff
@@ -54,20 +56,35 @@ function loadMusicVideo(){
 
 }
 
-function displayComments(){
-  
-  fetch('/data').then(response => response.json()).then((commentHistory) => {
+function displayComments(commentLimit){
+  if(typeof commentLimit === 'undefined'){
+      commentLimit = DEFAULT_COMMENT_LIMIT;
```

Comment:
The default behavior in the front end is to display 10 comments, but in the backend your default behavior is to display all comments. I'd pick one consistent default so that if something goes wrong on the frontend, you would still get consistent default behavior.